.bit/responses/5.6-Week 5 Step 6.md

```diff
@@ -0,0 +1,32 @@
+---
+files: n/a
+stepType: checks
+scripts: n/a
+week: 5
+step: 6
+name: Week 5 Step 6
+---
+Week 5 Step 6 ◯◯◯◯◯⬤ | 🕐 Estimated completion: 25 minutes
+
+## Documentation
+
+### ✅  Task:
+
+- [ ] Document your first component in blog.md
```

Comment:
Specify the "component" specifically

Reply:
Hey @emsesc can you clarify what you mean by this?

Reply:
@shreythecray Yup! I meant define what component you're talking about. Essentially, what is the component you're referring to?

Reply:
I'm not sure if I can do this here since they will be deciding on their own which component they will be working on